### PR TITLE
Update doc and prepare for CRAN

### PR DESCRIPTION
**Pull request by lionel-**
PTAL @hadley

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,60 +14,129 @@ Get the development version from github with:
...
 
-## Usage
 
-### Get installed font
+## Usage
 
-Get the path to an installed font `foo` with the `font_foo()` getters:
+### Fonts installed in the R library
+
+fontquiver is an interface to fonts installed in the R library. It
```
Comment (hadley) [inline README.md]: What is the R library? "fonts installed in R packages" might be better.

File: README.md [hunk 2]
```diff
@@ -14,60 +14,129 @@ Get the development version from github with:
...
+provides convenient and structured access, for instance, to the
+Bitstream Vera font family installed with the
+[fontBitstreamVera](https://cran.r-project.org/package=fontBitstreamVera)
+package, or the Liberation family that comes with
+[fontLiberation](https://cran.r-project.org/package=fontLiberation).
+The fonts too heavy to be distributed on CRAN can be accessed by
+fontquiver through Github-only packages such as
+[fontDejaVu](https://github.com/lionel-/fontDejaVu). These collections
+of font are referred to as `fontset` in fontquiver. Call
```
Comment (hadley) [inline README.md]: font -> fonts

Comment (hadley) [reply README.md]: Need an example of a fontset, e.g. Each package bundles a set of fonts (or `fontset`) which typically include bold and italic variations, but may also include...

File: README.md [hunk 3]
```diff
@@ -14,60 +14,129 @@ Get the development version from github with:
...
+platforms. Without fontquiver fonts, the SVGs generated by svglite
+would have slight differences depending on the versions of the system
+fonts used to compute text metrics.
+
+
+### Categories of fonts
+
+The standard categories of fonts in R are the `sans`, `serif`, and
+`mono` families and the `plain`, `italic`, `bold`, and `bolditalic`
+faces.However, font nomenclatures are extremely rich and go well
```
Comment (hadley) [inline README.md]: Missing space

File: cran-comments.md [hunk 4]
```diff
@@ -11,10 +7,14 @@ The package has been split so that it is covered by only one licence
 
 0 errors | 0 warnings | 1 note
 
-* This is a new release.
+Notes:
+
+* Changed maintainer email
+* Concern about fontLiberation's licence but if I'm not mistaken it is
```
Comment (hadley) [inline cran-comments.md]: You need to make a more definitive statement here

**Comment by lionel-** [comment]
Comments addressed.

**Comment by hadley** [comment]
LGTM

**Pull request merged**